README.md

```diff
@@ -1,4 +1,33 @@
 # Federal Grant Reporting Project
+Improving the experience of federal grant reporting.
+
+# Single audit resolution
+Enabling simpler, faster, easier, better resolution of single audit findings by
+agencies and grantees alike.
+
+## Local installation
+
+This is a Django app designed to run on Python 3.7.3.
+[pipenv](https://pipenv.readthedocs.io) is recommended for creating a virtual
+environment and managing dependencies. If you don't already have `pipenv`
+installed:
+
+1. Confirm that your `pip` uses Python 3, not Python 2, then
+2. Install pipenv by running `pip install pipenv`.
+
+Now you can prepare your development environment by running:
+
+```
+git clone git@github.com:18F/federal-grant-reporting.git
+cd federal-grant-reporting
+cd single-audit
+pipenv shell
+pipenv install
+./manage.py runserver
```

Comment:
When I get to the `./manage.py runserver` step locally, the result is an error stack trace like this:

```
Traceback (most recent call last):
  File "./manage.py", line 15, in <module>
...
ModuleNotFoundError: No module named 'single_audit.settings'
```

Taking a look at the relevant source code, I see there's no `settings.py` in the single_audit folder.

Do you see the same error result when you run locally?

Reply:
I tried adding an empty `settings.py` file under `single_audit` and got some new messages:

```
...
    raise ImproperlyConfigured("The SECRET_KEY setting must not be empty.")
django.core.exceptions.ImproperlyConfigured: The SECRET_KEY setting must not be empty.
```

Will keep tinkering in this direction. 😄

Reply:
Here are the `settings.py` values I've found so far that Django needs to get started properly:

```
- SECRET_KEY
- DEBUG
- ROOT_URLCONF
```

Reply:
Whoops! It'll need a few more values too. Hold off on this for now; I'll add a sanitized `settings.py` file.

Reply:
@alexsoble Added! Ready for another look.

(Doesn't currently include instructions for setting up a Postgres database, but I can add those when I commit the models.)

Reply:
I re-ran with the new `settings.py` file and got: 

```
ModuleNotFoundError: No module named 'resolve_findings'
```

Opened a small patch to this PR to fix: #84.